### ISSUE TITLE
BUG - runs fail with error in int_zendesk__field_history_pivot

### ISSUE THREAD

**Issue by annichia** [fivetran/dbt_zendesk#56]
**Are you a current Fivetran customer?** 
Yes. Maria Riolo, Data Analyst at Zearn

**Describe the bug**

Upon running, we get the error:

```
Database Error in model int_zendesk__field_history_pivot (models/ticket_history/int_zendesk__field_history_pivot.sql)
  syntax error at or near "pivot"
  LINE 123: ), pivot as (
               ^
```

Our daily runs started failing Dec 10, 2021, but at the time we hadn't made any changes to our own code since Dec 3. 

**Steps to reproduce**

`dbt run -m int_zendesk__field_history_pivot`

(or just `dbt run`)

**Project variables configuration**

```yml

# Name your package! Package names should contain only lowercase characters
# and underscores. A good package name should reflect your organization's
# name or the intended use of these models
name: 'zearn_redshift'
version: 1.0.0
config-version: 2

# This setting configures which "profile" dbt uses for this project. Profiles contain
# database connection information, and should be configured in the  ~/.dbt/profiles.yml file
profile: 'zearn-redshift'

# These configurations specify where dbt should look for different types of files.
# The `source-paths` config, for example, states that source models can be found
# in the "models/" directory. You probably won't need to change these!
source-paths: ["models"]
analysis-paths: ["analysis"]
test-paths: ["tests"]
data-paths: ["data"]
macro-paths: ["macros"]

target-path: "target"  # directory which will store compiled SQL files
clean-targets:         # directories to be removed by `dbt clean`
  - "target"
  - "dbt_modules"

on-run-start:
  # create schemas so we don't run into errors with the on-run-end hooks
  # if only doing a partial build or e.g. just running dbt seed
  - "CREATE SCHEMA IF NOT EXISTS \"{{ target.schema }}_marts\""
  - "CREATE SCHEMA IF NOT EXISTS \"{{ target.schema }}_staging\""
  - "CREATE SCHEMA IF NOT EXISTS \"{{ target.schema }}_exports\""
  - "CREATE SCHEMA IF NOT EXISTS \"{{ target.schema }}_zendesk_staging\""
  - "CREATE SCHEMA IF NOT EXISTS \"{{ target.schema }}_ad_hoc\""

on-run-end:
  - "GRANT USAGE ON SCHEMA \"{{ target.schema }}_marts\" to group looker, group zearn_analytics, group zearn_product"
  - "GRANT USAGE ON SCHEMA \"{{ target.schema }}_exports\" to jitterbit_sfdc_export, group looker, group zearn_analytics, group zearn_product"
  - "GRANT USAGE ON SCHEMA \"{{ target.schema }}_staging\" to group looker, group zearn_analytics, group zearn_product"
  - "GRANT USAGE ON SCHEMA \"{{ target.schema }}_zendesk_staging\" to group looker, group zearn_analytics, group zearn_product"
  - "GRANT SELECT ON ALL TABLES IN SCHEMA \"{{ target.schema }}_marts\" TO group looker, group zearn_analytics, group zearn_product"
  - "GRANT SELECT ON ALL TABLES IN SCHEMA \"{{ target.schema }}_exports\" TO jitterbit_sfdc_export, group zearn_analytics"
  - "GRANT SELECT ON ALL TABLES IN SCHEMA \"{{ target.schema }}_staging\" TO group zearn_analytics, group looker"
  - "GRANT SELECT ON ALL TABLES IN SCHEMA \"{{ target.schema }}_ad_hoc\" TO group zearn_analytics, group looker"
  - "GRANT SELECT ON ALL TABLES IN SCHEMA \"{{ target.schema }}_zendesk_staging\" TO group zearn_analytics, group looker"

# You can define configurations for models in the `source-paths` directory here.
# Using these configurations, you can enable or disable models, change how they
# are materialized, and more!

# In this example config, we tell dbt to build all models in the example/ directory
# as views (the default). These settings can be overridden in the individual model files
# using the `{{ config(...) }}` macro.
models:
  +bind: false

  zearn_redshift:
    +materialized: view
    staging:
      +materialized: view
      +schema: staging
      classification:
        +materialized: table
        support_tables:
          +materialized: table
      rostering:
        +materialized: table
      usage:
        +materialized: table
        +dist: user_id
        +sort: event_type, event_type_clean, created_at
    marts:
      +materialized: view
      +schema: marts
      summaries:
        +materialized: table
        +dist: user_id
        +sort: daydate
      exports:
        +materialized: table
        +schema: exports
    ad_hoc:
      +materialized: table
      +schema: ad_hoc
  zendesk:
    +schema: staging
    intermediate:
      +schema: staging
    sla_policy:
      +schema: staging
    ticket_history:
      +schema: staging
vars:
  school_years: ['2012-2013', '2013-2014', '2014-2015', '2015-2016', '2016-2017', '2017-2018', '2018-2019', '2019-2020', '2020-2021', '2021-2022']
  rostering_school_years: ['2016-2017', '2017-2018', '2018-2019', '2019-2020', '2020-2021', '2021-2022']
  random_seed: 3655122572989899184
  burbio_table_date: "2020-01-01" #default value needed to generate docs
  burbio_file_name: fake_file.csv #default value needed to generate docs
  district_student_info_district_id: 0 #default value needed to generate docs
  zendesk_source:
    zendesk_database: production
    zendesk_schema: ft_zendesk

```

**Package Version**

```yml
packages:
  - package: dbt-labs/dbt_utils
    version: [">=0.3.0"]
  - package: fivetran/zendesk
    version: [">=0.7.0"]
```

**Warehouse**

 
- [ ] BigQuery
- [x] Redshift
- [ ] Snowflake
- [ ] Postgres
- [ ] Databricks
- [ ] Other (provide details below)

**Please indicate the level of urgency** 
We do use our zendesk data. It's not an emergency right now, but it would be a big help to get it resolved before the end of the week so that our nightly syncs will be up and running over our winter break.

**Are you interested in contributing to this package?** 

- [ ] Yes, I can do this and open a PR for your review.
- [ ] Possibly, but I'm not quite sure how to do this. I'd be happy to do a live coding session with someone to get this fixed. 
- [x] No, I'd prefer if someone else fixed this.  I don't have the time and/or don't know what the root cause of the problem is.

**Comment by fivetran-joemarkiewicz** [comment]
Hi @annichia thanks so much for opening this issue!

this is actually an issue we saw come up in another Redshift package user a few days ago. The next release of the Zendesk package (this week) will rename the `pivot` CTE to `pivots` so this issue will no longer happen for Redshift users. I will post back here once we have merged and released the fix.

**Comment by fivetran-joemarkiewicz** [comment]
Hey @annichia 👋 

Thanks again for opening this issue. I wanted to share some news that this fix have been applied to the latest `v0.7.1` release of the zendesk package! Feel free to upgrade to the latest version and you should see the error be resolved.

Feel free to open another issue if you encounter any other bugs or want to chat more about this ticket. Thanks again!
